### ISSUE TITLE
Add aria-labellebdy attribute to the draggable balloon

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/balloons-and-static-electricity#192]
In #190 , @terracoda said
>"Yellow balloon" and Green balloon are not announced when they are grabbed. I think they need to be.[...]Are we using a an aria-labledby attribute for the aria-live regions?

I will add an aria-labelledby attribute to the balloon headings so that they are read when the balloon is picked up.  We are not using an aria-labelledby attribute for the live regions since these are more generally used for all alerts in any sim, and are not specific to the balloon.

**Comment by jessegreenberg** [comment]
This is a little bit dependent on the discussion we have concerning #183.

**Comment by jessegreenberg** [comment]
Marking as high priority, I think this should be done for #188.

**Comment by jessegreenberg** [comment]
aria-labelledby has been added.
